faucet.go

```diff
@@ -947,18 +886,25 @@ func (l *lightningFaucet) generateInvoice(homeTemplate *template.Template,
 func (l *lightningFaucet) payInvoice(homeTemplate *template.Template,
 	homeState *homePageContext, w http.ResponseWriter, r *http.Request) {
 
-	// Disable pay invoice if user set this parameter
-	if l.disablePayInvoices {
-		http.Error(w, "invoices payment was disabled", 403)
-		return
+	// Verify IP before open a channel
```

Comment:
Previously the code checked if the feature was disabled **before** checking the client timelimit, I think this should be maintained.